NativeScript/runtime/MetadataBuilder.mm

```diff
@@ -810,6 +810,14 @@
     Class klass = objc_getClass(containingClass.c_str());
     // TODO: Find out if the isMethodCallback property can be determined based on a UITableViewController.prototype.viewDidLoad.call(this) or super.viewDidLoad() call
 
+    NSString* message = [NSString stringWithFormat:@"MetadataBuilder::InvokeMethod class {%s}, selector {%s}, isInitializer {%s}, type {%s}, lib {%s}",
+                         containingClass.c_str(),
+                         meta->selectorAsString(),
+                         meta->isInitializer() ? "true":"false",
+                         meta->typeName(),
+                         meta->topLevelModule()->getName()];
+    tns::LogDebugMessage(message.UTF8String);
```

Comment:
```
if(debug) {
   NSString* message = [NSString stringWithFormat:@"MetadataBuilder::InvokeMethod class {%s}, selector {%s}, isInitializer {%s}, type {%s}, lib {%s}",
                         containingClass.c_str(),
                         meta->selectorAsString(),
                         meta->isInitializer() ? "true":"false",
                         meta->typeName(),
                         meta->topLevelModule()->getName()];
    tns::Log(message.UTF8String);
}
```

if you don't check for debug first, you'll waste processing on building the message and never show it

Reply:
Done